pkg/app/piped/platformprovider/kubernetes/kubernetes.go

```diff
@@ -32,6 +32,7 @@ const (
 	LabelIgnoreDriftDirection = "pipecd.dev/ignore-drift-detection" // Whether the drift detection should ignore this resource.
 	LabelSyncReplace          = "pipecd.dev/sync-by-replace"        // Use replace instead of apply.
 	AnnotationConfigHash      = "pipecd.dev/config-hash"            // The hash value of all mouting config resources.
+	AnnotationIndex           = "pipecd.dev/index"                  // The index of resource used to sort them before using.
```

Comment:
@khanhtc1202 @knanao  Still finding a better name. If you guys have any better ideas please let me know.

Reply:
Is `pipecd.dev/priority` more suitable?
Because Index basically seems to be unique but in this context, it needs to not be so. but it's up to you.

Reply:
how about: order, ordinal, priority ?

Reply:
`order` looks nice. Let me apply it.

Reply:
Addressed!